.gitignore

```diff
@@ -3,3 +3,6 @@ phpunit.xml
 composer.lock
 composer.phar
 vendor/*
+
+# IDEs
+/.idea/
```

Comment:
This should never be in the gitignore. They should be in your global gitignore file.